src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java

```diff
@@ -93,8 +96,7 @@ public void testCreateCardReturnsCard() {
   public void testCreateCardThrowsException() {
     when(accountService.getAccountById(accountId)).thenReturn(account);
     when(userService.getUserByLogin(user.getLogin())).thenReturn(null);
-    assertThrows(CardArgumentException.class,
-        () -> cardService.createCard(cardDto),
-        "No such user");
+    assertThrows(
+        CardArgumentException.class, () -> cardService.createCard(cardDto), "No such user");
```

Comment:
Please, reformat code

Reply:
Sorry, I forgot to fix this place) 
Fixed.